contrib/devtools/circular-dependencies.py

```
@@ -6,7 +6,7 @@
 import sys
 import re
 import multiprocessing
-from typing import Dict, List, Set
+from multiprocess import Pool # type: ignore[import]
```

Flake8 (7.3.0)

[error] 'multiprocess.Pool' imported but unused

(F401)